matrix/docker/query_runner.py

```diff
@@ -1,9 +1,81 @@
 """Script to pull from sqs and run redshift queries. Will be dockerized."""
+import json
+import os
 
+from matrix.common.aws.sqs_handler import SQSHandler
+from matrix.common.aws.s3_handler import S3Handler
+from matrix.common.aws.batch_handler import BatchHandler
+from matrix.common.aws.redshift_handler import RedshiftHandler
+from matrix.common.config import MatrixInfraConfig
+from matrix.common.logging import Logging
+from matrix.common.request.request_tracker import RequestTracker, Subtask
 
-def main():
-    pass
+logger = Logging.get_logger(__name__)
+
+
+class QueryRunner:
+
+    def __init__(self):
+        self.sqs_handler = SQSHandler()
+        self.s3_handler = S3Handler(os.environ["MATRIX_QUERY_BUCKET"])
+        self.batch_handler = BatchHandler()
+        self.redshift_handler = RedshiftHandler()
+        self.matrix_infra_config = MatrixInfraConfig()
+
+    @property
+    def query_job_q_url(self):
+        return self.matrix_infra_config.query_job_q_url
+
+    @property
+    def query_job_deadletter_q_url(self):
+        return self.matrix_infra_config.query_job_deadletter_q_url
+
+    def run(self, max_loops=None):
+        loops = 0
+        while True and (max_loops is None or loops < max_loops):
```

Comment:
Is the first `True` for anything?

Reply:
Yea, this loop will execute forever if no max_loops param is passed in. Essentially this means that each ecs query runner long polls sqs every 15 seconds into eternity.  The max_loops is only set from the test currently

Reply:
Ohhhh, I see what you are saying. You're right in the True is doing nothing haha. Will remove

Reply:
Thanks for the explanation regardless!